clients/pkg/promtail/wal/writer.go

```diff
@@ -247,7 +247,6 @@ func (ew *entryWriter) WriteEntry(entry api.Entry, wl WAL, _ log.Logger) error {
 
 	var fp uint64
 	lbs := labels.FromMap(util.ModelLabelSetToMap(entry.Labels))
-	sort.Sort(lbs)
```

Comment:
All sort calls have been removed because:

* The stringlabels `labels.Labels` doesn't implement `sort.Interface`
* Labels are always sorted now